demo/docbot/cluster_bootstrap.py

```diff
@@ -168,43 +171,64 @@ def initialize_connector(client: MLClient) -> None:
         print("Connector 'Cohere Connector' already exists. Skipping initialization.")
 
 
-def initialize_model(client: MLClient) -> None:
+def initialize_model(client: MLClient, model_name: str, model_descp: str) -> None:
     """
+    Initialize a model in OpenSearch.
```

Comment:
We need a comment here to specify that when using text-embed-2 we do not need to specify parameters but if using something else we need to add parameters.